tests/app/views/test_search_queries.py

```diff
@@ -184,39 +178,28 @@ def test_escaped_characters():
 
 
 @pytest.fixture(scope='module', autouse=True)
-def dummy_services():
+def dummy_services(services_mapping_definition):
     """Fixture that indexes a bunch of fake G-Cloud services so that searching can be tested."""
 
-    # Create a context where get_services_mapping has been patched, despite that fixture being function-
-    # scoped - see commentary below.
-    with contextlib.contextmanager(tests.conftest.services_mapping)():
```

Comment:
I'm not really sure what's going on here, but the new version definitely looks nicer :+1: